# For timestamps
Header header

# Debug
bool isEnabled
float64 walking_state
float64 td_force
float64 to_force
float64 td_position
